src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -1054,6 +1054,9 @@ public function testAlmostCircular($visibility)
 
         $container = new $container();
 
+        $logger = $container->get('monolog.logger');
+        $this->assertEquals(new \stdClass(), $logger->handler);
```

Comment:
ContainerBuilderTest also has testAlmostCircular method that should be synced with this one

Reply:
Fixed